page/login.jsx

```diff
@@ -14,12 +14,45 @@ css(`
 }
 `)
 
-
 export const Login = () => (
   <Div class="div-login">
-    <P fg="comment">{logo.split("#")}</P>
-    <NavLink href={`${API}/link/github`}  style={{fontSize: 30}}>
-      Join with Github
+    <P fg="comment">{logo.split('#')}</P>
+
+    <P fg="comment">
+      {'\n'}
+      <strong>NB : </strong>
+      you should have a github and discord account first.{'\n'}
+      If you don't have one, please create them before connecting.{'\n'} create
+      a github account{' '}
+      <NavLink
+        style={{ cursor: 'pointer' }}
+        target="_blank"
+        href="https://www.github.com"
+      >
+        https://www.github.com
+      </NavLink>
+      {'\n'} create a discord account{' '}
+      <NavLink
+        style={{ cursor: 'pointer' }}
+        target="_blank"
+        href="https://www.discord.com"
+      >
+        https://www.discord.com
+      </NavLink>
+    </P>
+    <br />
+    <NavLink href={`${API}/link/github`} style={{ fontSize: 30 }}>
+      <button
+        style={{
```

Comment:
it's better to put here the className of you  button instead to write a big object style for the reading

Reply:
ok i will do it